### ISSUE TITLE
[Feature request] AirPod support

### ISSUE THREAD

**Issue by winterflags** [folivoraAI/BetterTouchTool#955]
Is there any chance that AirPod input could be used with BTT?

Specifically, I would like to bind the double-tap on the AirPod to a function on the connected computer.

In macOS, it's possible to assign AirPod double-tap to Play/Pause Music. But for some reason *this only works in iTunes*. If you use the Play/Pause button on the Mac keyboard, or on the wired EarPods, it works differently because the command can be intercepted by other applications. I use a Chrome extension that intercepts the Play/Pause button and it will do so on popular websites like Soundcloud and Youtube.

TL;DR: Is there any way to add functionality to support AirPod sending key commands in BTT?

**Comment by fifafu** [comment]
Unfortunately I haven't found a way to get notified when the double-tap happens :(

**Comment by sidewinder12s** [comment]
Damn, I was really hoping you might be able to map more taps to different actions like muting Microphones in apps.
